examples/model_interpreter.py

```diff
@@ -0,0 +1,81 @@
+#  Copyright (c) Microsoft Corporation.
+#  Licensed under the MIT License.
+
+
+import qlib
+from qlib.config import REG_CN
+
+from qlib.utils import exists_qlib_data, init_instance_by_config
+from qlib.tests.data import GetData
+
+market = "csi300"
+benchmark = "SH000300"
+
+###################################
+# config
+###################################
+data_handler_config = {
```

Comment:
This dict appears too many times.
Will it be better to put it in `qlib.test.config.py`?